recipes/syned/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "syned" %}
+{% set version = "1.0.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5c07ef5fb1212af0702c1b66bd0390e5f7938637e8f6a8c2041b9bf8c7e2128e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python
+    - scipy
+    - setuptools
+  run:
+    - numpy
+    - python
+    - scipy
+    - setuptools
+
+test:
+  imports:
+    - syned
+    - syned.beamline
+    - syned.beamline.optical_elements
+    - syned.beamline.optical_elements.absorbers
+    - syned.beamline.optical_elements.crystals
+    - syned.beamline.optical_elements.gratings
+    - syned.beamline.optical_elements.ideal_elements
+    - syned.beamline.optical_elements.mirrors
+    - syned.examples
+    - syned.storage_ring
+    - syned.storage_ring.magnetic_structures
+    - syned.util
+    - syned.widget
+
+about:
+  home: https://github.com/lucarebuffi/syned
+  license: GNU General Public v3 or later (GPLv3+)
```

From https://spdx.org/licenses/:
```suggestion
  license: GPL-3.0-or-later
```